### DISCUSSION TITLE
Does anyone know what the use of 'move vae to gpu' is?

### DISCUSSION THREAD

**Discussion by zhibisora** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#138] [Q&A] [answered]
Does anyone know what the use of 'move vae to gpu' is? It doesn't seem to be mentioned in the help document.

**Comment by Kahsolt** [comment, marked as answer]
For those guys who enabled `--lowvram` or `--midvram` in startup comandline (though you are highly recommended NOT to do so when you have tiled-diffusion), their VAE might be transfered to CPU to leave VRAM space for other models. In this case they have to enable this option for more efficiency.

Now this option is automatically enabled, should not worry in most cases.
